core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -165,6 +167,20 @@ public SqlStatements setSqlLogger(SqlLogger sqlLogger) {
         return this;
     }
 
+    public boolean getAllowUnusedBindings() {
```

Comment:
Typically boolean getters get an `is` prefix instead of `get`

Reply:
Personally I hate that convention because it often doesn't make any grammatical sense... not every boolean is an `is` state and singular... What would you name the methods? `areUnusedBindingsAllowed` and `unusedBindingsAreAllowed(bool)`?

Reply:
Could make it grammatical e.g. `isUnusedBindingAllowed()`

Reply:
setter `unusedBindingIsAllowed(bool)` then?